### ISSUE TITLE
Towns won't revert when deleted – empty plot-block-data folder

### ISSUE THREAD

**Issue by jacobtilly** [TownyAdvanced/Towny#3589]
### What steps will reproduce the problem?
(0. Activate revert-on-unclaim using config or /tw, if not already activated)
1. Create a town
2. Build/destroy something
3. Delete the town
4. Watch it _not_ revert

Furthermore, a trip into the Towny/data/plot-block-data folder reveals total emptiness...

### What is the expected output?
The expected output would be for the chunks to revert to before they were claimed. I'd also expect the plot-block-data folder not to be empty.

### What is your Towny version?
Version 0.95.2.0.

### What is your spigot/craftbukkit version?
MC 1.15.1, CraftBukkit version git-Spigot-492a779-83af86d.

### Please use Pastebin.com to link the following files:

**Skipping these files is not an option. It is the easiest way to diagnose an issue ticket properly. If you do skip this step then your ticket will most likely be deleted and you will be asked to resubmit.**
1. Your full server startup from the logs\latest.log : https://pastebin.com/jwDb61XN
2. Your Towny config.yml (if using MySQL please remove password,) : https://pastebin.com/yJgzivZf
3. Your townyperms.yml : https://pastebin.com/wEEwUNED
4. Your log's error : No errors shown. Everything relevant in towny.log is
```
2019-12-28 17:51:49,498 [Server thread] INFO com.palmergames.bukkit.towny - [Global Msg] lunkebo created a new town called Testtown
2019-12-28 17:55:07,952 [Server thread] INFO com.palmergames.bukkit.towny - [Global Msg] Staden Testtown gick under!
```

(optional) If this is to do with permissions, 
5. Your permissions file : N/A

(optional) If this is to do with chat,
6. Your chatconfig.yml : N/A
7. Your channels.yml : N/A

**Comment by suneettipirneni** [comment]
I can confirm I have also seen this happen this should be fixed as soon as possible

**Comment by LlmDl** [comment]
Should be fixed in 0.94.2.5

**Comment by LlmDl** [comment]
https://github.com/TownyAdvanced/Towny/commit/7b5183594bbdc4525d8576ed5c89415641d971e7#diff-4e1c21fa1515514d7bc4b37abb9dc484R1706-R1727 Was the culprit here.
